components/api_request_helper/api_request_helper.h

```diff
@@ -11,15 +11,19 @@
 #include <string>
 
 #include "base/callback.h"
+#include "base/callback_helpers.h"
 #include "base/containers/flat_map.h"
 #include "net/traffic_annotation/network_traffic_annotation.h"
+#include "services/data_decoder/public/cpp/json_sanitizer.h"
 #include "url/gurl.h"
 
 namespace network {
 class SharedURLLoaderFactory;
 class SimpleURLLoader;
 }  // namespace network
 
+namespace data_decoder {}
```

Comment:
This should be removed, there is no forward declaration in it

Reply:
done